.github/workflows/notebooks.yml

```
@@ -41,12 +41,16 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Download data file from OSF
+      - name: Download data from OSF
         run: |
           mkdir ${TEST_DATA_HOME}
           pip install osfclient
           osf -p 39k4x fetch hcpdata.npz "${TEST_DATA_HOME}/hcpdata.npz"
 
+      - name: Download SynthStrip model
+        run: |
+          wget https://github.com/freesurfer/freesurfer/blob/dev/mri_synthstrip/synthstrip.1.pt -P "${TEST_DATA_HOME}/synthstrip.1.pt"
```

This is downloading to `/home/runner/nifreeze-tests//synthstrip.1.pt/synthstrip.1.pt`

```suggestion
          wget https://github.com/freesurfer/freesurfer/blob/dev/mri_synthstrip/synthstrip.1.pt -P "${TEST_DATA_HOME}"
```

---

Push-forced in 0135322. Thanks.